src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped working!");
```

Comment:
Similarly, for the stop message, consider rephrasing it to: 'Excavator has stopped working!'. This will make the message more informative.